widget/ui/src/components/SwapDetail/SwapDetail.tsx

```diff
@@ -1,106 +1,73 @@
-import type { PendingSwap } from '../../containers/History/types';
```

Comment:
We are going to make the `ui` standalone and it needs to be clean from `PendingSwap`, Could we move `widget/ui/src/components/SwapDetail` to `widget/embedded/src/components/SwapDetail`?